src/validate_test.py

```diff
@@ -1,9 +1,10 @@
+from config import DEFAULT_YEAR, SUPPORTED_YEARS
 from language import Language, DEFAULT_LANGUAGE
-from validate import parse_accept_language, DEFAULT_YEAR, SUPPORTED_YEARS
+from validate import parse_accept_language
 
-SUPPORTED_LANGUAGES = (Language.ENGLISH.lang_code, Language.JAPANESE.lang_code)
+SUPPORTED_LANGUAGES = (Language.en.lang_code, Language.ja.lang_code)
```

Comment:
Should these be EN, JA?

Reply:
Yup. Fixed.

We really should write more of these test cases. Only discovered this file recently!